mlflow/java/scoring/src/main/java/org/mlflow/sagemaker/MLeapPredictor.java

```diff
@@ -55,17 +55,37 @@ public MLeapPredictor(String modelDataPath, String inputSchemaPath) {
   @Override
   protected PredictorDataWrapper predict(PredictorDataWrapper input)
       throws PredictorEvaluationException {
-    PandasRecordOrientedDataFrame pandasFrame = null;
+    PandasSplitOrientedDataFrame pandasFrame = null;
     try {
-      pandasFrame = PandasRecordOrientedDataFrame.fromJson(input.toJson());
+      pandasFrame = PandasSplitOrientedDataFrame.fromJson(input.toJson());
     } catch (IOException e) {
       logger.error(
-          "Encountered a JSON conversion error during conversion of Pandas dataframe to LeapFrame.",
+          "Encountered a JSON parsing error during conversion of input to a Pandas Dataframe"
```

Comment:
nit: DataFrame

Reply:
Fixed the capitalization throughout the Java scoring server code.